### PR TITLE
Snapcraft Build recipe added

### PR DESCRIPTION
**Pull request by sahaRatul**
Enables creation of snap package which can be installed on snap enabled linux distros like Ubuntu.

https://snapcraft.io/cataclysm

### CODE REVIEW

File: data/cataicon.svg [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (BevapDin) [inline data/cataicon.svg]: Is the image in [data/xdg/cataclysm-dda.svg](https://github.com/CleverRaven/Cataclysm-DDA/tree/master/data/xdg/cataclysm-dda.svg) not good? Why do we need two icon files?

Comment (sahaRatul) [reply data/cataicon.svg]: Definitely missed that. Let me modify and commit.

**Comment by kevingranade** [comment]
Side note, please use https://cataclysmdda.org as the project site.

**Comment by ZhilkinSerg** [comment]
I need to press any key after `snap run cataclysm` to show main screen. Is it intended?

**Comment by sahaRatul** [comment]
@ZhilkinSerg This doesn't happen on stable 0.C build, only for daily builds. I have no idea why.

**Pull request merged**